api/models.py

```diff
@@ -68,30 +68,53 @@ def __str__(self):
 
 class Item(models.Model):
     """Stores all items"""
+
+    AVAILABLE = "Available"
+    ALLOCATED = "Allocated"
+    LOST = "Lost"
+    DAMAGED = "Damaged"
+
+    item_statuses = (
+        (AVAILABLE, "Available"),
+        (ALLOCATED, "Allocated"),
+        (LOST, "Lost"),
+        (DAMAGED, "Damaged")
+    )
+
     item_code = models.CharField(max_length=50, blank=True)
     serial_number = models.CharField(max_length=50, blank=True)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified = models.DateTimeField(auto_now=True, editable=False)
-    assigned_to = models.ForeignKey('User', blank=True,
+    assigned_to = models.ForeignKey('User',
+                                    blank=True,
+                                    default=1,
                                     on_delete=models.PROTECT)
     model_number = models.ForeignKey(ItemModelNumber, null=True,
                                      on_delete=models.PROTECT)
+    status = models.CharField(max_length=9,
+                              choices=item_statuses,
+                              default="Available")
 
     def clean(self):
         if not self.item_code and not self.serial_number:
             raise ValidationError(('Please provide either the serial number,\
                                asset code or both.'), code='required')
 
+        elif self.status in self.item_statuses:
```

Comment:
One last change: since `self.item_statuses` is a tuple of paired tuples you might need `dict(self.item_statuses)` for this to work.

Reply:
Done.